tasks/main.yml

```diff
@@ -38,21 +38,35 @@
     mode: 0644
   notify: restart youtrack service
 
-- name: download youtrack jar file
+- name: download youtrack zip file
   ansible.builtin.get_url:
-    url: "https://download.jetbrains.com/charisma/youtrack-{{ youtrack_version }}.jar"
-    dest: "{{ youtrack_releases_dir }}/youtrack-{{ youtrack_version }}.jar"
-    checksum: "sha256:https://download.jetbrains.com/charisma/youtrack-{{ youtrack_version }}.jar.sha256"
+    url: "https://download.jetbrains.com/charisma/youtrack-{{ youtrack_version }}.zip"
+    dest: "{{ youtrack_releases_dir }}/youtrack-{{ youtrack_version }}.zip"
+#    checksum: "sha256:https://download.jetbrains.com/charisma/youtrack-{{ youtrack_version }}.zip.sha256"
     owner: "{{ youtrack_user }}"
     group: "{{ youtrack_group }}"
     mode: 0640
 
+- name: Install Unzip
+  apt:
+    name: unzip
+    update_cache: yes
+
+- name: Unarchive zip file
+  ansible.builtin.unarchive:
+    src: "{{ youtrack_releases_dir }}/youtrack-{{ youtrack_version }}.zip"
+    dest: "{{ youtrack_releases_dir }}/"
+    remote_src: yes
+
+- name: Mkdir bin
+  shell: mkdir -p /home/youtrack/bin
```

Comment:
`ansible.builtin.file` but what's the reason to create `bin` directory and not to use script from youtrack distribution?